src/lib/fonts/fonts-dialog.rb

```diff
@@ -862,6 +876,33 @@ def widgets
       widgets_description
     end
 
+    # create copy of system settings; remove fonts-config generated
+    # config files to have such fontconfig setup as fonts-config 
+    # wouldn't never run; point fontconfig to this configuration
+    def create_default_fontconfig
+      family_list_file = FontsConfigCommand::local_family_list_file
+      rendering_file = FontsConfigCommand::rendering_config
+      metric_bw_symlink = FontsConfigCommand::metric_compatibility_bw_symlink
+      metric_file = FontsConfigCommand::metric_compatibility_config
+      metric_symlink = FontsConfigCommand::metric_compatibility_symlink
```

Comment:
ruby style guide said that for accessors or method it is preferred to use `.` instead of `::`